### ISSUE TITLE
live_grep does not search in node_modules with custom rg cmd

### ISSUE THREAD

**Issue by morhaham** [ibhagwan/fzf-lua#959]
### Info

- Operating System: macos ventura 13.5
- Shell: zsh
- Terminal: iterm2
- `nvim --version`: NVIM v0.9.2
- `fzf --version`: 0.33.0

<details>
<summary>fzf-lua configuration</summary>

```lua
return {
  "ibhagwan/fzf-lua",
  -- optional for icon support
  dependencies = { "nvim-tree/nvim-web-devicons" },
  keys = {
    {
      "<leader>ff",
      "<cmd>lua require('fzf-lua').files({ fd_opts = '--color=never --type f --hidden --follow --no-ignore' })<CR>",
      desc = "Find files",
    },
    {
      "<leader><leader>",
      "<cmd>lua require('fzf-lua').git_files()<CR>",
      desc = "Find git files",
    },
    {
      "<leader>fb",
      '<cmd>lua require("fzf-lua").buffers()<CR>',
      desc = "Find buffers",
    },
    {
      "<leader>fq",
      '<cmd>lua require("fzf-lua").quickfix()<CR>',
      desc = "Find in quickfix list",
    },
    {
      "<leader>fl",
      '<cmd>lua require("fzf-lua").loclist()<CR>',
      desc = "Find in location list",
    },
    {
      "<leader>fa",
      '<cmd>lua require("fzf-lua").args()<CR>',
      desc = "Find in argument list",
    },
    {
      "<leader>g",
      "<cmd>lua require('fzf-lua').live_grep({ cmd = 'rg --color=always --smart-case --no-ignore' })<CR>",
      desc = "live grep",
    },
  },
}

```
</details>

### Description 
I want to grep in `node_modules`.
With this setup live grep ignores `.gitignore` except for `node_modules`.
Take a look at my mapping for live_grep at the bottom of the config. 
I have also tried to launch it manually with:
`:lua require'fzf-lua'.live_grep({ cmd = "rg --color=always --smart-case --no-ignore --glob '!.git/**' --glob '!node_modules/**'" })`

Same result, live_grep ignores .gitignore except for node_modules.

**Comment by ibhagwan** [comment]
Fzf-lua display the results coming back from `rg`, whatever the result will be in the shell should be displayed within fzf-lua, solve the issue with your command in the shell and the same will happen with fzf-lua.

Can you add `debug=true` to your mapping? That will display the rg command at the first line of output, run the same command in the shell and find out why there is no results within node_modules:
```lua
:lua require'fzf-lua'.live_grep({ debug = true, cmd = "rg --color=always --smart-case --no-ignore --glob '!.git/**' --glob '!node_modules/**'" })
```

**Comment by ibhagwan** [comment]
Also, with  `--glob '!.git/**' --glob '!node_modules/**'` you're specifically telling `rg` to ignore anything starting with `.git|node_modules`  so why would these be displayed, did you mean without the exclmation  mark `--glob '.git/**' --glob 'node_modules/**'`?

**Comment by morhaham** [comment]
> Also, with `--glob '!.git/**' --glob '!node_modules/**'` you're specifically telling `rg` to ignore anything starting with `.git|node_modules` so why would these be displayed, did you mean without the exclmation mark `--glob '.git/**' --glob 'node_modules/**'`?

Right! with that change it is working and searching in node_moduels too. I have 2 questions:
1. How can I copy the debug line (or any result line text) as it is too long to see the whole command, and when selected it won't open the buffer with the debug line.
2. Why it is required to glob specifically `node_modules`? why the `--no-ignore` flag is not enough? 

Thank you for this amazing plugin btw.

<img width="1135" alt="image" src="https://github.com/ibhagwan/fzf-lua/assets/3867713/64131a94-23e7-44dd-bcd5-6d7ded1e2307">

**Comment by ibhagwan** [comment]
> How can I copy the debug line (or any result line text) as it is too long to see the whole command, and when selected it won't open the buffer with the debug line.

Since there is no line wrap in fzf you need to make it so there is enough space on screen to see it, what I do is either close the preview (F4) or rotate it clockwise (F6), if I still need more space I go full screen (F2), if that isn’t enough I would expand my tmux pane to take the entire screen space (tmux-prefix-Z).

> Why it is required to glob specifically node_modules? why the --no-ignore flag is not enough?
as stated in ripgrep docs:

Technically `--no-ignore` should be enough, play around with it in the shell and if it’s still not working as you expect open an issue with the `rg` repo (as fzf-lua doesn’t affect this functionality).

> so basically --no-ignore is an alias to -uuu. -uuu works for me but --no-ignore not.

That would have to be answered by the maintainer of rg :)

> Thank you for this amazing plugin btw.

Thank you for the kind words ❤️ 

Btw, I highly recommend updating to the latest version of fzf, 0.33 is pretty old by now, with the latest version you’ll get better UX, for example zero-reload staging changes in `git_status`, deleting buffers with `ctrl-x`, etc.

**Comment by ibhagwan** [comment]
Closing, feel free to reopen if you need more help.
